components/partnerLogoContainer.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div>
-        <b-row v-for="(chunk, index) in team" :key="index">
-            <b-col v-for="(logo) in chunk" :key="logo" class="d-flex align-items-center justify-content-center">
+        <b-row v-for="(chunk, index) in team" :key="'team' + index">
```

Comment:
no muy importante pero me daba errors en la console

Reply:
Si hacemos esto no nos saldrían los logos, los chunks son bloques:
```js
chunks: [
    [{img}, {img}, {img}, {img}, {img}],
    [{img}, {img}, {img}, {img}, {img}],
    [{img}, {img}, {img}, {img}, {img}]
...
];
```

Pero lo hemos resuelto en https://github.com/rescue-app/rescueapp.es/commit/7929f0beee4375b8ff99e0f271ca3258c9566536

Reply:
Nada... que no salían los cambios enteros sorry, toda la razón con tu commit

Reply:
mmm esto tambien funcionaba (solo modificaba las keys) pero si ya lo habéis cambiado ahi lo quito!